site/src/components/TemplateLayout/TemplateLayout.tsx

```diff
@@ -152,65 +140,59 @@ export const TemplateLayout: FC<PropsWithChildren> = ({ children }) => {
           }
         >
           <Stack direction="row" spacing={3} className={styles.pageTitle}>
-            {!isLoading && (
```

Comment:
I'm not sure why we were wrapping all these blocks in `isLoading` - seems like we just need to make sure the template is there (and we do on line 114).